content/agent/proxy.md

```diff
@@ -29,22 +29,66 @@ A few options are available to send traffic to Datadog over SSL/TLS for hosts th
 
 Traditional web proxies are supported natively by the Agent. If you need to connect to the Internet through a proxy, edit your Agent configuration file.
 
-##### Agent v6
+### Agent v6
+
+Set different proxy servers for `https` and `http` requests in your Agent `datadog.yaml` configuration file. 
+The Agent uses `https` to send data to Datadog, but integrations might use `http` to gather metrics. No matter the proxied queries, you can activate SSL on your proxy server. Below are some configuration examples for your `datadog.yaml` file:
+
+Setting an HTTP proxy for all `https` queries:
+
+```
+proxy:
+    https: http://<SSL_PROXY_SERVER_FOR_HTTPS>:<PORT>
+```
 
-Edit the `datadog.yaml` file with your proxy information. Use the `no_proxy` list to specify hosts that should bypass the proxy.
+Setting an HTTPS proxy for both `https` and `http` queries:
 
 ```
 proxy:
-    http: http://user:password@proxy_for_http:port
-    https: http://user:password@proxy_for_https:port
-#   no_proxy:
-#     - host1
-#     - host2
+    https: https://<SSL_PROXY_SERVER_FOR_HTTPS>:<PORT>
+    http: https://<SSL_PROXY_SERVER_FOR_HTTP>:<PORT>
 ```
 
+Setting a `<USERNAME>` and `<PASSWORD>` to contact the proxy server for both `https` and `http` queries:
+
+```
+proxy:
+    https: http://<USERNAME>:<PASSWORD>@<PROXY_SERVER_FOR_HTTPS>:<PORT>
+    http: http://<USERNAME>:<PASSWORD>@<PROXY_SERVER_FOR_HTTPS>:<PORT>
+```
+
+Using the `no_proxy` list to specify hosts that must bypass the proxy:
+
+```
+proxy:
+    https: http://<USERNAME>:<PASSWORD>@<PROXY_SERVER_FOR_HTTPS>:<PORT>
+    http: http://<USERNAME>:<PASSWORD>@<PROXY_SERVER_FOR_HTTPS>:<PORT>
+    no_proxy:
+      - host1
+      - host2
+```
+
+#### Proxy with environment variables
+
+Starting with Agent v6.4, you can set your proxy settings through environment variables:
+
+* `DD_PROXY_HTTPS`: Sets a proxy server for `https` queries.
+* `DD_PROXY_HTTP`: Sets a proxy server for `http` queries.
+* `DD_NO_PROXY`: Sets a list of hosts that should bypass the proxy. The list is space-separated.
+
+Environment variables have precedence over values in the `datadog.yaml` file. If the environment variables are present with an empty value (e.g. ``DD_HTTP_PROXY=""``), the Agent uses those empty values instead of lower-precedence options.
+
+On Unix hosts, a system-wide proxy might be specified using standard environment variables, such as `HTTPS_PROXY`, `HTTP_PROXY`, and `NO_PROXY`. The Agent uses these if present. Be careful, as such variables also impact every request from integrations, including orchestrators like Docker, ECS, and Kubernetes.
+
+The Agent uses the following values in order of precedence:
+
+1. `DD_PROXY_HTTPS`, `DD_PROXY_HTTP`, and `DD_NO_PROXY` environment variables
+2. `HTTPS_PROXY`, `HTTP_PROXY`, and `NO_PROXY` environment variables
+3. Values inside `datadog.yaml`
+
 [Refer to our log collection documentation page to learn how to forward your logs with a proxy][7].
```

Comment:
for clarity I'd move this line up above the `#### Proxy with environment variables` title (or even at the very beginning of the section). Don't have a strong opinion though

Reply:
References are usually at the end of a section. :)